crypto.go

```diff
@@ -461,3 +468,177 @@ func DeriveSigningKey(seed []byte, salt []byte, iter int) (*[32]byte, *[64]byte)
 	copy((*publicKey)[:], privateKeyBytes[32:])
 	return publicKey, privateKey
 }
+
+// EncryptFile encrypts the contents of the file plainFileName using the ak and stores the ciphertext in encryptedFileName
+func EncryptFile(plainFileName string, encryptedFileName string, ak SymmetricKey) (int, string, error) {
+	hasher := md5.New()
+	var dk []byte
+	dkSymmetric := RandomSymmetricKey()
+	dk = dkSymmetric[:]
+	edk, edkN := SecretBoxEncryptToBase64(dk[:], ak)
+	headerText := fmt.Sprintf("%v.%s.%s.", FILE_VERSION, edk, edkN)
+	headerBytes := []byte(headerText)
+	hasher.Write(headerBytes)
+	// create and open encryptedFileName
+	tempFile, err := os.Create(encryptedFileName)
+	if err != nil {
+		return 0, "", err
+	}
+	// write the header to the encrypted file
+	size, err := tempFile.Write(headerBytes)
+	if err != nil {
+		return 0, "", err
+	}
+	readFile, err := os.Open(plainFileName)
+	if err != nil {
+		return 0, "", err
+	}
+	defer func() {
+		if err = readFile.Close(); err != nil {
+			fmt.Println("Error closing file: ", err)
```

Comment:
Thank about the user experience when a developer sees this random line, add some helpful context like
`e3db-clients-go:EncryptFile: error closing file %s`